astroid/brain/brain_typing.py

```diff
@@ -140,6 +157,150 @@ def infer_typing_typevar_or_newtype(node, context_itton=None):
     return node.infer(context=context_itton)
 
 
+def infer_typing_newtype(
+    node: nodes.Call, context_itton: typing.Optional[context.InferenceContext] = None
```

Comment:
Let's use `context` instead of `context_itton` here, that's the standard in most other similar functions. I think `context_itton` is a legacy name.

Reply:
Switched to `ctx`. Avoids a collision with the module-level `from astroid import context`, and seems to be the convention for the rest of the module.